pkg/localmetrics/localmetrics.go

```diff
@@ -89,16 +93,28 @@ var (
 
 // UpdateAWSMetrics updates the total AWS Accounts metric every N hours
 func UpdateAWSMetrics(kubeClient kubeclientpkg.Client, hour int) {
-	awsClient, _ := awsclient.GetAWSClient(kubeClient, awsclient.NewAwsClientInput{
+	metricLogger := log.WithValues("Namespace", "aws-account-operator-operator")
+
+	awsClient, err := awsclient.GetAWSClient(kubeClient, awsclient.NewAwsClientInput{
 		SecretName: account.AwsSecretName,
 		NameSpace:  awsv1alpha1.AccountCrNamespace,
 		AwsRegion:  "us-east-1",
 	})
 
+	if err != nil {
+		metricLogger.Error(err, "Failed to get awsClient")
+		return
+	}
+
 	d := time.Duration(hour) * time.Minute
```

Comment:
Please change this to the correct time unit.

Reply:
Done